src/components/backend-ai-webui.ts

```diff
@@ -1862,9 +1862,9 @@ export default class BackendAIWebUI extends connect(store)(LitElement) {
                 </div>
                 <div slot="navigationIcon" class="vertical-line" style="height:20px;margin:0;"></div>
                 <div class="horizontal center layout" slot="title" id="welcome-message" style="font-size:12px;margin-left:10px;" class="draggable">
-                  <p>${_t('webui.menu.WelcomeMessage')}</p>
-                  <p class="user-name">${this._getUsername()}</p>
```

Comment:
please check  https://github.com/lablup/backend.ai-webui/pull/1008/commits/94bfc08edeaedca389560d4d4726f66cb9ddea80 commit related to `user-name` class.